docs/src/pages/docs/installation/index.mdx

```diff
@@ -100,3 +100,11 @@ username: admin
 ```bash
 password: admin
 ```
+
+
+### 5. Connecting your local database instance to superset
+
+When running superset using `docker` or `docker-compose` it runs in its own docker container, as if the superset was running in a seperate machine entirely. Therefore attempts to connect to your local database with hostname `localhost` won't work as `localhost` refers to the docker container superset is running in, and not your actual host machine. Fortunately docker provides an easy way to access network resources in the host machine from inside a container, we will leverage this capability to connect to our local database instance. Here the instructions are for connecting to postgresql (which is running on your host machine) from superset (which is running in its docker container). Other databases may have slightly different configurations but gist would be same and boils down to 2 steps -
+
+1. Configuring the local postgresql/database instance to accept public incoming connections. By default postgresql only allows incoming connections from localhost only, but re-iterating once again, `localhosts` are different for host machine and docker container. For postgresql this involves make one-line changes to the files `postgresql.conf` and `pg_hba.conf`, you can find helpful links tailored to your OS / PG version on the web easily for this task.
```

Comment:
It might be an OS thing. In linux first of all `host.docker.internal` does not work, and second it was necessary when used with `172.18.0.1` as hostname. Though, agreed that this is no more than a hack. Maybe we can add a *note* here for convenience but those managing production databases should already know better!

Reply:
Ah yes so maybe it'd be better to split this up. I think what it is, is that on mac you can do `host.docker.interal` and don't need to touch your postgresql confs. The behavior is described [here](https://docs.docker.com/docker-for-mac/networking/#there-is-no-docker0-bridge-on-macos).

On other OS you don't have the magic dns entry and so you need to go through the bridge network IP which likely requires the config updates.